### DISCUSSION TITLE
How can I get rid of this?

### DISCUSSION THREAD

**Discussion by babakfp** [gitkraken/vscode-gitlens#3045] [Q&A] [answered]
![Screenshot 2023-12-08 105713](https://github.com/gitkraken/vscode-gitlens/assets/44144724/24e008a1-1d6d-444a-b075-802b136da4c3)

**Comment by eamodio** [comment, marked as answer]
That is a feature of VS Code itself and not GitLens. You can [open an issue](https://github.com/microsoft/vscode/issues/new?assignees=&labels=&template=bug_report.md) on the VS Code repository.
